docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -20,7 +20,10 @@ Supported load balancers
 Round robin
 ^^^^^^^^^^^
 
-This is a simple policy in which each healthy upstream host is selected in round robin order.
+This is a simple policy in which each healthy upstream host is selected in round
+robin order. If weights are assigned to endpoints in a locality, then a weighted
```

Comment:
as long as you have to merge master and move release note, maybe some links to v2 fields here for weight, etc.?